### ISSUE TITLE
Mouse pointer abnormally large under KDE Plasma 6

### ISSUE THREAD

**Issue by liamhays** [kovidgoyal/kitty#7201]
**Describe the bug**
I think this bug is similar if not identical to #4860. I upgraded my Arch Linux laptop to Plasma 6 this morning and the I-beam-shaped cursor that appears when I hover over the kitty window is abnormally large compared to other windows. I have Plasma global scaling set to 100%, though I had set it to 125% to test and then restored it to 100%. This issue persists after ending a Plasma session and starting it again.

**To Reproduce**
1. Open kitty
2. Hover over the window

**Screenshots**
Top image is kitty with mysterious large cursor, bottom image is Kate with normal sized cursor. Both windows are the exact same size.
![image](https://github.com/kovidgoyal/kitty/assets/5673976/437902de-f1fd-4264-b5c1-1e399242e3f4)
![image](https://github.com/kovidgoyal/kitty/assets/5673976/7884b644-f01d-49bd-b174-8eae6ee416ac)

**Environment details**
```
kitty 0.31.0 created by Kovid Goyal
Linux [hostname] 6.7.8-arch1-1 #1 SMP PREEMPT_DYNAMIC Sun, 03 Mar 2024 00:30:36 +0000 x86_64
Arch Linux 6.7.8-arch1-1 (/dev/tty)

Running under: Wayland
Frozen: False
Paths:
  kitty: /usr/bin/kitty
  base dir: /usr/lib/kitty
  extensions dir: /usr/lib/kitty/kitty
  system shell: /bin/bash
Loaded config files:
  /home/liam/.config/kitty/kitty.conf

Config options different from defaults:
enable_audio_bell       False
font_family             JetBrains Mono
touch_scroll_multiplier 4.0

Important environment variables seen by the kitty process:
	PATH                                /home/liam/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/var/lib/flatpak/exports/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/liam/.local/bin
	LANG                                en_US.UTF-8
	EDITOR                              nano
	SHELL                               /bin/bash
	DISPLAY                             :0
	WAYLAND_DISPLAY                     wayland-0
	USER                                liam
	XDG_CONFIG_DIRS                     /home/liam/.config/kdedefaults:/etc/xdg
	XDG_CURRENT_DESKTOP                 KDE
	XDG_DATA_DIRS                       /home/liam/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/usr/local/share/:/usr/share/
	XDG_MENU_PREFIX                     plasma-
	XDG_RUNTIME_DIR                     /run/user/1000
	XDG_SEAT                            seat0
	XDG_SESSION_CLASS                   user
	XDG_SESSION_ID                      1
	XDG_SESSION_TYPE                    wayland
	XDG_VTNR                            1

```

**Additional context**
The problem is reproducible with `kitty --config NONE`.

**Comment by kovidgoyal** [comment]
dup of #6914

**Comment by liamhays** [comment]
The changelog shows that #6914 was fixed in version 0.32.0, but the Arch repos still have 0.31.0. Hopefully they can update it soon.

**Comment by gluonparticle** [comment]
This is actually an **Accessibility** feature offered by Plasma. To fix this:

**System Settings > Accessibility > Shake Cursor > Uncheck the box for "Shake cursor to find it"** 
You can set the magnification level as well;

**Comment by michaldybczak** [comment]
I have this issue on Tuxedo OS, with Ubuntu base 04.2024 and kernel 6.11.

Which package is responsible for this bug? @liamhays, you said "The changelog shows that https://github.com/kovidgoyal/kitty/issues/6914 was fixed in version 0.32." Version of which package?

Ubuntu base is always giving me a headache with its static packages with frozen bug ;(.

I don't have this problem on Manjaro, since this is patched a long, long time ago.

Operating System: TUXEDO OS 3
KDE Plasma Version: 6.2.0
KDE Frameworks Version: 6.6.0
Qt Version: 6.7.2
Kernel Version: 6.11.0-107009-tuxedo (64-bit)
Graphics Platform: Wayland
Processors: 16 × AMD Ryzen 7 7840HS w/ Radeon 780M Graphics
Memory: 30.6 GiB of RAM
Graphics Processor: AMD Radeon Graphics
Manufacturer: TUXEDO
Product Name: TUXEDO Sirius 16 Gen1

**Comment by liamhays** [comment]
> Which package is responsible for this bug? @liamhays, you said "The changelog shows that https://github.com/kovidgoyal/kitty/issues/6914 was fixed in version 0.32." Version of which package?

Version 0.32 of kitty. It turned out that I was only experiencing this issue because the Arch repos were abnormally slow to get 0.32 after it was released.
